airbyte-webapp/src/views/Connector/ConnectorCard/ConnectorCard.tsx

```diff
@@ -108,7 +111,8 @@ export const ConnectorCard: React.FC<ConnectorCardCreateProps | ConnectorCardEdi
           props.successMessage || (saved && props.isEditMode && <FormattedMessage id="form.changesSaved" />)
         }
       />
-      {job && <JobItem job={job} />}
+      {/* Show the job log only if advanced mode is turned on or the actual job failed (not the check inside the job) */}
+      {job && (advancedMode || !job.succeeded) && <JobItem job={job} />}
```

Comment:
Would it make any sense to present the "check connection" job as "Failed" if the check inside the job was not successful? 

To me, a "check connection" job is "unsuccessful" if either
1. The job itself failed to complete for some reason (e.g. could not connect to temporal), or
2. The check that is performed in the job failed (since that is the whole point of the check connection job)

I think it is useful to differentiate between these cases by having different failure messages/reasons shown, but I still consider both situations as a "failed check". So I feel that we should only show the "check connection" job as "successful" if it ran to completion and the check itself was successful. What do you think?

Reply:
Given that we hide it by default now from the UI at all, I feel we should keep it as successful even if the actual result of the check was negative (which we still show as negative, but the user won't see the successful Job run anymore unless they are in advanced mode). Given that it's a technical detail in advanced mode, I feel more inclined to not mess with the actual job status, and have that represent what happened to the actual technical job under the hood instead of trying to rewrite this. Also in case of the job being successful but the configuration check failed the logs won't have any error inside, which makes it a bit confusing if we'd show the job now as failed, but only have "successful" logs insight.

Reply:
Hmm okay I think I'm convinced that it's fine to leave it as is then. My only other suggestion is that maybe we could add a note or a tooltip somewhere in the successful job case that explains that even though the job was successful, that doesn't necessarily mean the actual check was successful